docs/arcaflow/workflows/expressions.md

```diff
@@ -28,31 +70,256 @@ $.foo.bar
+
+The mathematical expression 2<sup>3</sup> is represented in the expression language as `2^3`, which would output the integer `8`.
+
+### Equals
+
+The `==` equality operator checks for equality between the left and right type. It returns true when the left and right match.
+The operator currently supports the types `integer`, `float`, `string`, and `boolean`. If another type is required, please create an issue with the expected behavior of the operator with the needed type.
+
+For example, `2 == 2` results in `true`, and `"a" == "b"` results in `false`.
+
+### Not Equals
+
+The `!=` operator is the inverse of the [==](#equals ) operator. It returns true when the values do not match.
+The operator currently supports the types `integer`, `float`, `string`, and `boolean`. If another type is required, please create an issue with the expected behavior of the operator with the needed type.
+
+For example, `2 != 2` results in `false`, and `"a" != "b"` results in `true`.
+
+### Greater Than
+
+The `>` inequality operator outputs `true` if the left side is greater than the right side, and `false` otherwise. The operator requires numerical or string input.
+
+For example, the expression `3 > 3` would output the boolean `false`, and `4 > 3` would output `true`.
+
+### Less Than
+
+The `<` inequality operator outputs `true` if the left side is less than the right side, and `false` otherwise. The operator requires numerical or string input.
+
+For example, the expression `3 < 3` would output the boolean `false`, and `1 < 2` would output `true`.
+
+### Greater Than or Equal To
+
+The `>=` inequality operator outputs `true` if the left side is greater than or equal to the right side, and `false` otherwise. The operator requires numerical or string input.
+
+For example, the expression `3 >= 3` would output the boolean `true`, `3 >= 4` would output `false`, and `4 >= 3` would output `true`.
+
+### Less Than or Equal To
+
+The `<=` inequality operator outputs `true` if the left side is less than or equal to the right side, and `false` otherwise. The operator requires numerical or string input.
+
+For example, the expression `3 <= 3` would output the boolean `true`, `3 <= 4` would output `true`, and `4 <= 3` would output `false`.
+
+### Logical AND
+
+The `&&` operator returns `true` if both the left and right sides are `true`, and `false` otherwise. This operator requires boolean input.
+Note: There is no short-circuiting as it's currently implemented. Both the left and right are evaluated before the comparison takes place.
+
+All cases:
+| Left    | Right   | `&&`    |
+| ------- | ------- | ------- |
+| `true`  | `true`  | `true`  |
+| `true`  | `false` | `false` |
+| `false` | `true`  | `false` |
+| `false` | `false` | `false` |
+
+### Logical OR
+
+The `&&` operator returns `true` if either or both of the left and right sides are `true`, and `false` otherwise. It is a non-exclusive or operator. This operator requires boolean input.
```

Comment:
"non-exclusive or operator" reads like a (bizarre) phrase. You should probably set off the "or" as `or`.

Reply:
I removed that part.